apps/datahub/src/index.html

```diff
@@ -6,11 +6,8 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
-      rel="stylesheet"
-    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
can you explain what it is for ?

Reply:
It's what google recommends to use for google fonts:
![image](https://user-images.githubusercontent.com/10629150/144494533-572b0a0f-08a4-419a-bc0a-dfc00ebdd926.png)

Notice the two `preconnect` links, which work a bit like `preload` except they [only open a connection](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preconnect).

Reply:
Also, to explain a bit more, I chose to "hardcode" this in the datahub template in anticipation of people using google fonts; ideally this should be added dynamically and not rely on modifications in each app's template